src/java.base/share/classes/java/nio/MappedByteBuffer.java

```diff
@@ -397,6 +397,8 @@ public final MappedByteBuffer rewind() {
      * {@code force()} on the returned buffer, will only act on the sub-range
      * of this buffer that the returned buffer represents, namely
      * {@code [position(),limit())}.
+     *
+     * @since 17
      */
     @Override
     public abstract MappedByteBuffer slice();
```

Comment:
This and the other 3 methods on which this `@since 17` is being added here were introduced in Java 17, through https://bugs.openjdk.org/browse/JDK-4833719. Before that change, these methods would have been available on this `MappedByteBuffer` through the super `ByteBuffer` class and those methods have been specified to return a `ByteBuffer`. After that change in JDK-4833719, these methods now return a `MappedByteBuffer` which is an observable change. So adding `@since 17` to these methods looks correct to me.